common/js/options.js

```diff
@@ -133,30 +180,32 @@ document.addEventListener("DOMContentLoaded", function() {
     });
 
     save_button.addEventListener("click", function () {
+    	var msg = "";
```

Comment:
It used to put the 'error' messages in separate alerts. Now, all are in one alert.